src/main/scala/org/bytedeco/sbt/javacpp/Platform.scala

```diff
@@ -20,7 +17,28 @@ object Platform {
    */
   val current: Seq[String] = sys.props.get(platformOverridePropertyKey) match {
     case Some(platform) if platform.trim().nonEmpty => platform.split(' ')
-    case _ => Seq(Loader.getPlatform)
+    case _ =>
```

Comment:
What's the advantage of *not* pulling in JavaCpp at the sbt level and re-using the loader?

Reply:
I chose this style, so I didn't have to think about whether the potential difference of the libraries would cause any trouble. It should work either way, though.